cdc/cdc/http_handler.go

```diff
@@ -39,8 +39,8 @@ const (
 	APIOpVarChangefeedID = "cf-id"
 	// APIOpVarTargetCaptureID is the key of to-capture ID in HTTP API
 	APIOpVarTargetCaptureID = "target-cp-id"
-	// APIOpVarTableID is the key of table ID in HTTP API
-	APIOpVarTableID = "table-id"
+	// APIOpVarKeySpanID is the key of keyspan ID in HTTP API
+	APIOpVarKeySpanID = "keyspan-id"
```

Comment:
what's the relationship between keyspan id and region id?

Reply:
In the current implementation, keyspan and region have a one-to-one relationship. But keyspan id is not equal to region id, keyspan id only represents an interval [start, end). keyspan-id = hash(start+end).